cli/tests/transfer.rs

```diff
@@ -355,7 +356,7 @@ fn test_transfer_multisession_signing() {
         &rpc_client,
         &CliConfig::recent_for_tests(),
         &offline_fee_payer_signer.pubkey(),
-        sol_to_lamports(1.0) + 2 * fee,
+        sol_to_lamports(1.0) + 2 * fee_two_sig,
```

Comment:
is this correct? it says `2 * fee_two_sig` which would be 4?!

Reply:
it's original test logic :) I only rename the `fee` to `fee_two_sig` for consistency. It checks `2 * fee_two_sig` at ln 368 below, feels intended.

Reply:
read the test, it sign a two-signature transaction twice that transfers `42.0` sol. `offline_fee_payer_signer` is one of the signer (eg payer) that is initiated with `1 + 2 * fee_two_sig`. After submitted transaction, it's balance should be `original_balance - fee_for_two_sig_transaction = (1 + 2 * fee_two_sig) - fee_two_sig = 1 + fee_two_sig`. Which it asserts at Ln#470: 
```
    check_balance!(
        sol_to_lamports(1.0) + fee_two_sig,
        &rpc_client,
        &offline_fee_payer_signer.pubkey(),
    );
```

So you are right that `offline_fee_payer_signer` does not need to be initialized with `2 *`. But the test is still valid, even a bit of confusing. Happy to straight it out separately.